packages/app/components/Settings/CourseAdminPanel.tsx

```diff
@@ -0,0 +1,98 @@
+import {
+  BellOutlined,
+  EditOutlined,
+  QuestionCircleOutlined,
+} from "@ant-design/icons";
+import { Col, Menu, Row, Space, Tooltip } from "antd";
+import { useRouter } from "next/router";
+import React, { ReactElement, useState } from "react";
+import styled from "styled-components";
+import { useProfile } from "../../hooks/useProfile";
+import CourseOverrideSettings from "./CourseOverrideSettings";
+import { SettingsPanelAvatar } from "./SettingsSharedComponents";
+import TACheckInCheckOutTimes from "./TACheckInCheckOutTimes";
+
+export enum CourseAdminOptions {
+  CHECK_IN = "CHECK_IN",
+  OVERRIDES = "OVERRIDES",
+}
+
+interface CourseAdminPageProps {
+  defaultPage: CourseAdminOptions;
+  courseId: number;
+}
+
+const VerticalDivider = styled.div`
+  @media (min-width: 767px) {
+    border-right: 1px solid #cfd6de;
+    margin-right: 32px;
+  }
+`;
+
+const CenteredText = styled.p`
+  text-align: center;
+`;
+
+export default function CourseAdminPanel({
+  defaultPage,
```

Comment:
lgtm, except that this `defaultPage` prop and the router stuff can probably be removed if you're not gonna let them be publicly linked.